.github/packager/packager.sh

```
@@ -4,19 +4,43 @@
+# --------------------------------------------------------------------------
+# CRITICAL FIX: Universal Root Directory Calculation (Runs from anywhere)
+# --------------------------------------------------------------------------
+if [ -n "$AERIUM_ROOT" ]; then
+    # Option 1: Use the root path provided by the user via the environment variable.
+    ROOT_DIR="$AERIUM_ROOT"
+    echo "Using custom root from \$AERIUM_ROOT: $ROOT_DIR"
+else
+    # Option 2: Fallback to calculating the root based on the script's location
+    # Assumes script is in /path/to/repo/.github/packager
+    SCRIPT_PATH="$( cd "$( dirname "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )"
+    ROOT_DIR=$(dirname $(dirname "$SCRIPT_PATH"))
```

Shellcheck (0.11.0)

[warning] Quote this to prevent word splitting.

(SC2046)